### PR TITLE
0x32 Origin 

### PR DESCRIPTION
**Pull request by mmsc2**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
 
Origin opcode was not implemented

Issue Number: 104

## What is the new behavior?

This PR adds the origin opcode to the kakarot VM, It currently fetches starknet address, It has to be changed in the future to fetch EVM address from the starknet address.

### CODE REVIEW

**Comment by danilowhk** [comment]
If you run "make format" should fix the "compilation-tests" issue

File: src/kakarot/instructions/environmental_information.cairo [hunk 1]
```diff
@@ -55,6 +56,39 @@ namespace EnvironmentalInformation {
...
+        
+        %{
+        import logging
+        logging.info("0x32 - ORIGIN")
+        %}
+
+        // Get Starknet address
+        /// TODO: get EVM address from Starknet address
+        let (tx_info) = get_tx_info();
+        let origin_address = Helpers.to_uint256(tx_info.account_contract_address);
```
Comment (LucasLvy) [inline src/kakarot/instructions/environmental_information.cairo:82]: update with the accounts registry 
```suggestion
       tx_info.account_contract_address -> evm address
```

**Review by LucasLvy — CHANGES_REQUESTED** [review]

File: src/kakarot/instructions/environmental_information.cairo (hunk 1 above)
Comment (mmsc2) [reply src/kakarot/instructions/environmental_information.cairo:127]: thank you for the feedback. I have been trying to import the function "get_evm_address" from the account_registry.cairo file but i wasn't able to do so, which would be the right way to use it?.

Comment (LucasLvy) [reply src/kakarot/instructions/environmental_information.cairo:127]: Check the balance opcode to see how to map the address

Comment (mmsc2) [reply src/kakarot/instructions/environmental_information.cairo]: Already fixed, thank you!

**Review by LucasLvy — APPROVED** [review]
Lgtm

**Pull request merged**